planetiler-core/pom.xml

```diff
@@ -138,6 +138,11 @@
       <artifactId>guava</artifactId>
       <version>31.1-jre</version>
     </dependency>
+    <dependency>
+      <groupId>org.geotools</groupId>
+      <artifactId>gt-geopkg</artifactId>
+      <version>${geotools.version}</version>
+    </dependency>
```

Comment:
I'm trying to eventually remove the dependency on geotools since it has licensing issues and the artifacts aren't in maven central (#126) - is there another library we could use instead that provides equivalent functionality?

Reply:
👍 wasn't aware of that, just went with it because `geotools` was already used.

There's http://ngageoint.github.io/geopackage-java/ (MIT License)

Reply:
Thanks, that looks promising! Also please update `NOTICE.md` with a blurb about the dependency you end up going with.